.env.example

```diff
@@ -0,0 +1,18 @@
+CMWL_PIPELINE_CONTAINER_PATH=/cmwl_pipeline/app
+CMWL_PIPELINE_CONTAINER_NAME=cmwl_pipeline
+CMWL_PIPELINE_CONTAINER_LOGS=/cmwl_pipeline/app/logs
+CMWL_DB_SERVER_NAME=172.17.0.1
+CMWL_DB_PORT_NUMBER=5432
```

Comment:
Apparently you should reuse this port in description of docker container for Postgres. I changed it to `5433` and application failed to start. Changing
```
    ports:
      - "5432:5432"
```
to
```
    ports:
      - ${CMWL_DB_PORT_NUMBER}:5432
```
in `docker-compose.yml` resolved this issue. I assume you should do the same thing for other variables like mongo DB port.

Reply:
Done